appveyor.yml

```diff
@@ -71,16 +71,12 @@ install:
           $env:path="$root;$root\Scripts;$root\Library\bin;$($env:path)"
     - cmd: conda config --set show_channel_urls true
     - cmd: conda update --yes --quiet conda
-    - cmd: set PYTHONUNBUFFERED=1
 
     - cmd: conda config --add channels conda-forge
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: obvci_install_conda_build_tools.py
```

Comment:
This line can probably go (xref: https://github.com/pelson/Obvious-CI/blob/master/scripts/obvci_install_conda_build_tools.py)